### DISCUSSION TITLE
can't use read py

### DISCUSSION THREAD

**Discussion by SerAlanTheBold** [neonbjb/tortoise-tts#503] [Q&A]
(tortoise-tts) C:\Users\Lylliana\anaconda3\tortoise-tts>python tortoise/read.py --textfile <DARKSOULSHonestMerchant.ansi> --voice leliana
Traceback (most recent call last):
  File "C:\Users\Lylliana\anaconda3\tortoise-tts\tortoise\read.py", line 37, in <module>
    with open(args.textfile, 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'leliana'

**Comment by fitbyprabhat** [comment]
same here @neonbjb

**Reply by LucidFir** [reply]
Get rid of the < and the >

**Comment by LucidFir** [comment]
Get rid of the < and the >

**Comment by neonbjb** [comment]
It's telling you the requested voice doesn't exist.
